core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -321,15 +321,19 @@ object AppType {
   case object Custom extends AppType {
     override def toString: String = "CUSTOM"
   }
+  case object CromwellOnAzure extends AppType {
```

Comment:
I decided a new `AppType` makes sense -- it is a different helm chart and a lot of the validation/handing is different for Cromwell vs CromwellOnAzure.